src/classes/HiFiCommunicator.ts

```diff
@@ -98,21 +102,24 @@ export class HiFiCommunicator {
      * @param transmitRateLimitTimeoutMS - User Data updates will not be sent to the server any more frequently than this number in milliseconds.
      * @param userDataStreamingScope - Cannot be set later. See {@link HiFiUserDataStreamingScopes}.
      * @param hiFiAxisConfiguration - Cannot be set later. The 3D axis configuration. See {@link ourHiFiAxisConfiguration} for defaults.
+     * @param raviSessionParams - Cannot be set later. The RAVI session parameters used when connecting to the API servers. See {@link RaviSessionParams} for details.
```

Comment:
I don't want to confuse customers with the "RAVI" name more than we have to. Let's call this something like "webrtcSessionParams" instead, and make the comment something like "Extra parameters used for configuring the underlying WebRTC connection to the API servers. These settings are not frequently used; they are primarily for specific jitter buffer configurations." (And then I'm OK with continuing to link to the RaviSessionParams docs.)